src/adaptation/Adaptation.jl

```diff
@@ -1,75 +1,59 @@
 module Adaptation
+export Adaptation
 
-import Base: string, rand
-using Random: GLOBAL_RNG, AbstractRNG
-using LinearAlgebra: Symmetric, UpperTriangular, mul!, ldiv!, dot, I, diag, cholesky, UniformScaling
-import LinearAlgebra, Statistics
-using ..AdvancedHMC: DEBUG
+using LinearAlgebra: LinearAlgebra
+using Statistics: Statistics
 using Parameters: @unpack, @pack!
 
-const AbstractScalarOrVec{T} = Union{T,AbstractVector{T}} where {T<:AbstractFloat}
+using ..AdvancedHMC: DEBUG, AbstractScalarOrVec
 
 abstract type AbstractAdaptor end
-
-##
-## Interface for adaptors
-##
-
-getM⁻¹(adaptor::T) where {T<:AbstractAdaptor} = error("`getM⁻¹(adaptor::$T)` is not implemented.")
-getϵ(adaptor::T) where {T<:AbstractAdaptor} = error("`getϵ(adaptor::$T)` is not implemented.")
-adapt!(
-    adaptor::T,
-    θ::AbstractVecOrMat{<:AbstractFloat},
-    α::AbstractScalarOrVec{<:AbstractFloat}
-) where {T<:AbstractAdaptor} = error("`adapt!(adaptor::$T, θ::AbstractVecOrMat{<:AbstractFloat}, α::AbstractScalarOrVec{<:AbstractFloat})` is not implemented.")
-reset!(adaptor::T) where {T<:AbstractAdaptor} = error("`reset!(adaptor::$T)` is not implemented.")
-initialize!(adaptor::T, n_adapts::Int) where {T<:AbstractAdaptor} = throw(MethodError(initialize!, adaptor, n_adapts))
-finalize!(adaptor::T) where {T<:AbstractAdaptor} = error("`finalize!(adaptor::$T)` is not implemented.")
+function getM⁻¹ end
+function getϵ end
```

Comment:
Are users supposed to work with or call these functions? IMO names of user-facing functions should not include fancy unicode characters. Additionally, a name such as `stepsize` or `getstepsize` would be more descriptive.

If it's not user-facing, it doesn't matter that much, but still more descriptive names might be easier to understand :man_shrugging:

Reply:
They are not exposed to users.